src/components/Documentation/Layout/SidebarMenu/index.tsx

```diff
@@ -31,26 +33,15 @@ const ICONS: { [key: string]: React.FC<{ className?: string }> } = {
 }
 
 interface ISidebarMenuItemProps {
-  children?: Array<{ label: string; path: string; source: boolean | string }>
-  label: string
-  path: string
-  source: boolean | string
+  item: INormalizedSidebarItem
   onClick: (isLeafItemClicked: boolean) => void
   activePaths?: Array<string>
-  type?: string
-  style?: string
-  icon?: string
 }
 
 const SidebarMenuItem: React.FC<ISidebarMenuItemProps> = ({
-  children,
-  label,
-  path,
+  item: { children, label, path, style, icon, type },
   activePaths,
-  onClick,
-  style,
-  icon,
-  type
+  onClick
```

Comment:
Slight signature tweak which makes more sense to me now that we have a type def for sidebar items.